Sources/App/Commands/Analyzer.swift

```diff
@@ -76,7 +77,7 @@ func analyze(application: Application, packages: [Package]) -> EventLoopFuture<V
                                                     withIntermediateDirectories: false,
                                                     attributes: nil)
         } catch {
-            let msg = "Failed to create checkouts directory: \(error.localizedDescription)"
+            let msg = "Failed to create checkouts directory: \(error)"
```

Comment:
I initially also used `error.localizedDescription` for reporting the `AppMetrics.push` error but noticed that that prints

```
The operation couldn’t be completed. (NIO.NIOConnectionError error 1.)
```

vs

```
NIOConnectionError(host: "127.0.0.1", port: 9091, dnsAError: nil, dnsAAAAError: nil, connectionErrors: 
[NIO.SingleConnectionFailure(target: [IPv4]127.0.0.1/127.0.0.1:9091, error: connection reset (error set): 
Connection refused (errno: 61))])
```

when it's `"\(error)"`. I wonder if we'd get a lot more error detail in our rollbar items this way, so I propose we give that a try.

Reply:
I've reverted this for now, because it also changes

```
[ ERROR ] ingestion error: missing Github API token [database-id: psql]
```

to

```
[ ERROR ] ingestion error: missingToken [database-id: psql]
```

for our on `LocalizedError`s. Best not mix this in here and look into this separately. In the past I've used https://swiftpackageindex.com/mxcl/LegibleError to get better error printouts and it might be a good idea to use that.